### PR TITLE
GRW-1746 / Feature / Household size input

### PR DESCRIPTION
**Pull request by alebedev**
## Describe your changes

- Nice custom field for household size
- Fixes for value interpolation in i18n

Alternative approach to dynamic fields PR here: https://github.com/HedvigInsurance/racoon/pull/651

## Justify why they are needed

To be closer to design vision for purchase forms

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Review by KarnellSchultz — APPROVED** [review]
lgtm

File: apps/store/src/services/PriceForm/data/SE_APARTMENT.ts [hunk 1]
```diff
@@ -19,10 +19,28 @@ export const SE_APARTMENT: Template = {
...
+            name: 'numberCoInsured',
+            label: { key: 'Household size' },
+            required: true,
+            defaultValue: 0,
+            max: 5,
+          },
+          layout: { columnSpan: 6 },
+        },
+      ],
+    },
```
Comment (gustaveen) [inline apps/store/src/services/PriceForm/data/SE_APARTMENT.ts:39]: Did we get feedback that we should put this as the second step? It the designs we have it as the last step 
Screenshot 2022-11-09 at 09 27 35

https://www.figma.com/file/rJZ1o8sR3YKhCyKA2BgucA/Hedvig.com-%C2%B7-Final-Wireframe-Designs?node-id=179%3A26186

File: apps/store/src/components/PriceForm/HouseholdSize.tsx [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+      />
+      <InputBase label={t(field.label.key)}>
+        {() => (
+          <Wrapper>
+            <Value>{t('HOUSEHOLD_SIZE_VALUE', { count: value })}</Value>
+            <StyledButton onClick={increment}>+</StyledButton>
+            <StyledButton onClick={decrement}>-</StyledButton>
+          </Wrapper>
+        )}
+      </InputBase>
```
Comment (gustaveen) [inline apps/store/src/components/PriceForm/HouseholdSize.tsx:43]: Oh, I think we already have an input like this you can use. `InputStepper` in the UI package :)

**Review by gustaveen — COMMENTED** [review]
Works great! 🙌 Just a comment on the input component

**Pull request merged**